lib/util/cleanup_url.dart

```diff
@@ -1,6 +1,6 @@
 /// Strips protocol, 'www.', and trailing '/' from [url] aka. cleans it up
 String cleanUpUrl(String url) {
-  var newUrl = url;
+  var newUrl = url.toLowerCase();
```

Comment:
While you're at it I would just go ahead and replace the whole function and let the `Uri` class handle the normalization instead of those awkward substrings. I'm thinking something of this sort:

```dart
String maybeUrlHost(String maybeUrl) =>
    urlHost(maybeUrl.startsWith('https://') ? maybeUrl : 'https://$maybeUrl');
``` 

And then changing the doc to the same as from `urlHost` but specifying that the passed string can be missing the `https://` prefix